k8s/upgrade/src/bin/upgrade-job/upgrade/utils.rs

```diff
@@ -29,6 +29,23 @@ pub(crate) async fn is_rebuilding(rest_client: &RestClientSet) -> Result<bool> {
         starting_token = volumes.next_token;
         for volume in volumes.entries {
             if let Some(target) = &volume.state.target {
+                let mut rebuild_count = 0;
+
+                for child in target.children.iter() {
+                    if child.rebuild_progress.is_some() {
+                        rebuild_count += 1;
```

Comment:
Can we not directly `return Ok(true)` from here or possibly break if the count increased from 0, no need of going on with the loop?

Also why is this needed if below check in line 49 is there? Is this for logging only?

Reply:
We need to print the logs as how many replica is rebuilding so we need to iterate over the loop.